src/cursor/sys/windows.rs

```diff
@@ -206,6 +224,33 @@ mod tests {
         assert_eq!(position().unwrap(), (0, 0));
     }
 
+    #[test]
+    fn test_move_to_next_line_winapi() {
+        move_to(0, 2).unwrap();
```

Comment:
Not necessarily in this PR, because other tests do the same thing. We should wrap all tests with a macro/fn which saves the cursor position, executes the test and then it restores the position. I can imagine that the current behaviour can create a mess from the tests output. Not saying that unit tests shouldn't do these things at all.

Reply:
We need to establish a baseline, we can't expect the cursor to be somewhere. I agree that those tests do more then they should to do and like the idea of those macro's. However, we can do those things in other PR's.